ui/package.json

```diff
@@ -30,15 +30,15 @@
     "@angular/platform-browser": "^5.2.2",
     "@angular/platform-browser-dynamic": "^5.2.2",
     "@angular/router": "^5.2.2",
-    "@clr/angular": "^0.11.2-patch",
-    "@clr/icons": "^0.11.2-patch",
-    "@clr/ui": "^0.11.2-patch",
+    "@clr/angular": "^0.11.12",
+    "@clr/icons": "^0.11.12",
+    "@clr/ui": "^0.11.12",
     "@swimlane/ngx-charts": "^7.1.1",
     "@types/geojson": "^1.0.6",
     "@webcomponents/custom-elements": "^1.0.0",
     "angular-2-local-storage": "^1.0.1",
     "angular2-highcharts": "^0.5.5",
-    "bootstrap": "^4.0.0-alpha.5",
+    "bootstrap": "4.0.0-alpha.5",
```

Comment:
Per Snyk, please change this 4.0.0 or higher

Reply:
It'll probably work with bootstrap 4, but the reason we have this is [Clarity UI requires _exactly_ this version](https://github.com/vmware/clarity/blob/ae0e115601472bbb9abdf956d19d4a9739ea698d/npm/clr-ui/package.json#L12)

Reply:
I opened an issue with Clarity b/c it seems weird to me that they should be pinned to exactly an alpha version.  This smells a lot like "no-one bothered updating our packaging data when bootstrap moved out of alpha."

Reply:
Luckily it would seem some good citizen already filed an issue against them: https://github.com/vmware/clarity/issues/2169